src/main/java/net/minestom/server/event/instance/InstanceSectionInvalidateEvent.java

```diff
@@ -0,0 +1,39 @@
+package net.minestom.server.event.instance;
+
+import net.minestom.server.event.trait.InstanceEvent;
+import net.minestom.server.instance.Instance;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This event is triggered when a section of an instance is manually marked as invalid.
+ * <p>
+ * Changes in this case are not known but indicate that its content must be reinterpreted.
+ */
+public class InstanceSectionInvalidateEvent implements InstanceEvent {
```

Comment:
Should mention that this should be triggered through Instance#invalidateSection rather than called manually (and maybe mark constructor internal additionally)